backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/workflow/workflowRestController.feature

```diff
@@ -0,0 +1,251 @@
+      | pickingCustomerLocation | picking_GLN | pickingCustomer          | true                | true         |
+
+    And metasfresh contains M_Inventories:
+      | M_Inventory_ID.Identifier | MovementDate | M_Warehouse_ID |
+      | workflowInventory         | 2021-10-12   | 540008         |
+    And metasfresh contains M_InventoriesLines:
+      | M_Inventory_ID.Identifier | M_InventoryLine_ID.Identifier | M_Product_ID.Identifier | QtyBook | QtyCount | UOM.X12DE355 |
+      | workflowInventory         | workflowInventoryLine         | workflowProduct_230822  | 0       | 1        | PCE          |
+    And complete inventory with inventoryIdentifier 'workflowInventory'
+    And after not more than 30s, there are added M_HUs for inventory
+      | M_InventoryLine_ID.Identifier | M_HU_ID.Identifier |
+      | workflowInventoryLine         | workflowProductHU  |
+
+    And metasfresh contains C_Orders:
+      | Identifier   | IsSOTrx | C_BPartner_ID.Identifier | DateOrdered |
+      | pickingOrder | true    | pickingCustomer          | 2021-04-15  |
+    And metasfresh contains C_OrderLines:
+      | Identifier       | C_Order_ID.Identifier | M_Product_ID.Identifier | QtyEntered |
+      | pickingOrderLine | pickingOrder          | workflowProduct_230822  | 1          |
+
+    And the order identified by pickingOrder is completed
+
+    And after not more than 30s, M_ShipmentSchedules are found:
+      | Identifier              | C_OrderLine_ID.Identifier | IsToRecompute |
+      | pickingShipmentSchedule | pickingOrderLine          | N             |
+
+    And create JsonWFProcessStartRequest for picking and store it in context as request payload:
+      | C_Order_ID.Identifier | C_BPartner_ID.Identifier | C_BPartner_Location_ID.Identifier |
+      | pickingOrder          | pickingCustomer          | pickingCustomerLocation           |
+
+    And the metasfresh REST-API endpoint path 'api/v2/userWorkflows/wfProcess/start' receives a 'POST' request with the payload from context and responds with '200' status code
+
+    And process response and extract picking step and main HU picking candidate:
+      | WorkflowProcess.Identifier | WorkflowActivity.Identifier | WorkflowStep.Identifier | WorkflowStepQRCode.Identifier |
+      | pickingWorkflow            | workflowPickingActivity     | workflowPickingStep     | workflowPickingQRCode         |
+    And create JsonPickingEventsList and store it in context as request payload:
+      | WorkflowProcess.Identifier | WorkflowActivity.Identifier | WorkflowStep.Identifier | WorkflowStepQRCode.Identifier | QtyPicked |
+      | pickingWorkflow            | workflowPickingActivity     | workflowPickingStep     | workflowPickingQRCode         | 1         |
+
+    And the metasfresh REST-API endpoint path 'api/v2/picking/events' receives a 'POST' request with the payload from context and responds with '200' status code
```

Comment:
I checked the mobile-webui-frontend code and there is no special http-request-headers set when calling the workflow endpoints. so I'm not sure exactly what you are referring to ... also checked the X-Api-Async in BE (thinking maybe this is what u had in mind) and by default it this is not set, the http call is handled synchronously (unless api_audit_config.isforceprocessedasync says otherwise)